### PR TITLE
Added scaffolding for the README

### PR DESCRIPTION
**Pull request by jasmingacic**
@hasheddan I've started adding stuff to the README. If you have any ideas please feel free to add.

Also I was not sure how the binary gets deployed and how to articulate that in the doc.

### CODE REVIEW

**Comment by hasheddan** [comment]
@jasmingacic I have opened #6 to track updating the test / build / publish pipeline. It may be worth setting up a call for this.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  hardware_reservation_id: next_available
+  locked: false
+  providerRef:
+    name: example
+    namespace: packet-infra-dev
+  reclaimPolicy: Delete
+```
+
+```
+$ kubectl craete -f device.yaml
```
Comment (displague) [inline README.md]: create

File: README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+```
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+For more details on Crossplane installation visit [here](https://crossplane.io/docs/v0.1/install-crossplane.html)
+
+## Installing `stack-packet` 
+
+* Create CRDs for the stack:
```
Comment (displague) [inline README.md:35]: With the [Crossplane CLI installed](https://github.com/crossplaneio/crossplane-cli#installation), the non-developer approach to installing the Stack would be:
 
```
kubectl crossplane stack install --cluster packethost/stack-packet
```

A requirement for offering this approach would be the existence of a stack-packet image at https://hub.docker.com/u/packethost

Comment (jasmingacic) [reply README.md:35]: @displague we are working on it.

**Pull request merged**